src/Squidex/app/features/rules/pages/rules/actions/email-action.component.html

```diff
@@ -0,0 +1,128 @@
+            </small>
+        </div>
+    </div>
+
+    <div class="form-group row">
+        <label class="col-3 col-form-label" for="username">Username</label>
+
+        <div class="col-9">
+            <sqx-control-errors for="port" [submitted]="actionFormSubmitted"></sqx-control-errors>
+
+            <input type="text" class="form-control" id="username" formControlName="username" />
+
+            <small class="form-text text-muted">
+                The username used to authenticate the sender.
+            </small>
+        </div>
+    </div>
+
+    <div class="form-group row">
+        <label class="col-3 col-form-label" for="password">Password</label>
+
+        <div class="col-9">
+            <sqx-control-errors for="password" [submitted]="actionFormSubmitted"></sqx-control-errors>
+
+            <input type="password" class="form-control" id="password" formControlName="password" />
+
+            <small class="form-text text-muted">
+                The password used to authenticate the sender.
+            </small>
+        </div>
+    </div>
+
+    <div class="form-group row">
+        <label class="col-3 col-form-label" for="from">From</label>
+
+        <div class="col-9">
+            <sqx-control-errors for="from" [submitted]="actionFormSubmitted"></sqx-control-errors>
+
+            <input type="text" class="form-control" id="from" formControlName="from" />
+
+            <small class="form-text text-muted">
+                The email is sent from?
+            </small>
+        </div>
+    </div>
+
+    <div class="form-group row">
+        <label class="col-3 col-form-label" for="to">To</label>
+
+        <div class="col-9">
+            <sqx-control-errors for="to" [submitted]="actionFormSubmitted"></sqx-control-errors>
+
+            <input type="text" class="form-control" id="to" formControlName="to" />
+
+            <small class="form-text text-muted">
+                The email will be sent to?
+            </small>
+        </div>
+    </div>
+
+    <div class="form-group row">
+        <label class="col-3 col-form-label" for="subject">Subject</label>
+
+        <div class="col-9">
+            <sqx-control-errors for="subject" [submitted]="actionFormSubmitted"></sqx-control-errors>
+
+            <input type="text" class="form-control" id="subject" formControlName="subject" />
+
+            <small class="form-text text-muted">
+                The subject line for this e-mail message.
+            </small>
+        </div>
+    </div>
+
+
+    <div class="form-group row">
+        <label class="col-3 col-form-label" for="body">Body</label>
+
+        <div class="col-9">
+            <sqx-control-errors for="body" [submitted]="actionFormSubmitted"></sqx-control-errors>
+
+            <textarea class="form-control" id="body" formControlName="body"></textarea>
+
+            <small class="form-text text-muted">
+                The message body
```

Comment:
If you have a look to the other hints you will that I have added a link to the docs for formatting action inputs. Can you add it too?